Dockerfile

```diff
@@ -38,7 +38,6 @@ RUN \
     # https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#repo-qg-apt-repo-manual-setup
     apt-get -q update && \
     apt-get -q install -y --no-install-recommends libmysqlclient-dev cmake golang-go python3-dev python3-pip python3-setuptools python3-wheel && \
-    pip3 install -r requirements.txt && \
```

Comment:
I'm not sure about the changes to the dockerfile here, so someone more familiar with the whole setup should take a look!